snippets/Google.Datastore.V1Beta3.Snippets/DatastoreDbSnippets.cs

```diff
@@ -660,7 +660,7 @@ public void GroupingQuery()
                 DistinctOn = { "type" },
                 Order =
                 {
-                    { "type", Direction.Ascending },
+                    "type", // If only the name is specified, it's implicitly ascending
                     { "priority", Direction.Ascending }
```

Comment:
You've left the "Direction.Ascending" here to show how to specify direction explicitly?